members/views/AccountCreate.py

```diff
@@ -0,0 +1,105 @@
+from django.urls import reverse
+from django.views.decorators.clickjacking import xframe_options_exempt
+from members.forms import signupForm
+from django.shortcuts import render
+from django.utils import timezone
+from django.http import HttpResponseRedirect
+from django.contrib.auth.models import User
+from members.models.family import Family
+from members.models.person import Person
+
+
+@xframe_options_exempt
+def AccountCreate(request):
+    if request.method == "POST":
+        # figure out which form was filled out.
+        if request.POST["form_id"] == "signup":
+            # signup has been filled
+            signup = signupForm(request.POST)
+            if signup.is_valid():
+                # check if family already exists
+                # TODO: rewrite this! >>>>
+                try:
+                    family = Family.objects.get(
+                        email__iexact=request.POST["parent_email"]
+                    )
+                    # family was already created - we can't create this family again
```

Comment:
Et forslag ville være at samle if sætningerne i en `and` der er short circuit i python så det kan stadig fungere uden at kaste en undtagelse hvis de første fejler. 
Man kunne så bruge `walrus` operatoren hvis man ville have assignment med, men det er måske ikke så kønt.

Reply:
@Rotendahl : Jeg synes det er fine forslag til refac. Men jeg mener ikke at det er en del af denne opgave. Grunden til at de er markeret "nye linier" er pga af at det er flyttet fra `members/views/EntryPage.py`